INSTALL.asciidoc

```diff
@@ -2,19 +2,30 @@
 pragmatic development environment on openSUSE
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-1.  install openQA package to get all the runtime dependencies
+1.  https://github.com/os-autoinst/openQA/blob/master/docs/Installing.asciidoc#installation[install the openQA package] to get all the runtime dependencies
 +
 ---------------------
 sudo zypper in openQA
 ---------------------
++
+Alternatively installing the following packages will satisfy all dependencies at time of writing
++
+-----------------------------------------------------------------------
+zypper qemu tesseract optipng opencv tesseract-ocr sqlite3 mumble     \
+       libtheora-devel opencv-devel fftw3-devel libsndfile-devel      \
+       pkg-config libtool autoconf automake gcc-c++ make perl-JSON    \
+       perl-Mojolicious perl-Mojolicious-Plugin-AssetPack             \
+       perl-IPC-System-Simple perl-Exception-Class perl-Test-ANSIColor\
+       perl-Test-Compile perl-Test-More perl-Test-Simple perl-XML     \
+       perl-carp perl-Class-Accessor perl-Net-SSH2 perl-Crypt-DES
+-----------------------------------------------------------------------
```

Comment:
I agree with @okurz to not explicitly add this list. We add new deps all the time.
There are `zypper if --requires os-autoinst` and `zypper if -t srcpackage --requires os-autoinst` calls you can use to get required capabilities for both building and running.